Sources/File.swift

```diff
@@ -188,14 +188,10 @@ extension File {
         return filedetach(file)
     }
 
-    public func close() -> Bool {
-        if closed {
-            return false
-        }
-
+    public func close() throws {
+        guard !closed else { throw ClosableError.alreadyClosed }
```

Comment:
you mean if it has some data? i think it should flush data on close, and only rethrow flush errors.